### PR TITLE
Add 'DebugPackage' model for ddeb packages

### PR DESCRIPTION
**Pull request by pieterlexis**
These can be used to represent Ubuntu Bionic's ".ddeb" debug packages. These can be added to distributions like normal deb packages.

Note that this feature also requires changes to the syncing task. Opened for early review and comments.

ref #6757

### CODE REVIEW

File: pulp_deb/app/serializers/content_serializers.py [hunk 1]
```diff
@@ -488,7 +513,7 @@ def deferred_validate(self, data):
         """Validate for InstallerPackage."""
         data = super().deferred_validate(data)
 
-        if data.get("section") != "debian-installer":
+        if data.get("section") != "debian-installer" or data.get("package_type") == "ddeb":
```
Comment (mdellweg) [inline pulp_deb/app/serializers/content_serializers.py:516]: Again "package_type" != "udeb"

**Comment by quba42** [comment]
@pieterlexis You could exchange the `ref #6757` with `fixes #6757` in your commit message (since that is what you are aiming for here). I have removed the pulpbot comment  from this PR, so this should also associate the ticket with this PR automatically.

Let me know if you need any help with this PR. I do appreciate the effort!

**Comment by pieterlexis** [comment]
> @pieterlexis You could exchange the `ref #6757` with `fixes #6757` in your commit message (since that is what you are aiming for here). I have removed the pulpbot comment from this PR, so this should also associate the ticket with this PR automatically.

Ack, will do.

> Let me know if you need any help with this PR. I do appreciate the effort!

Probably later. I just rebased it today, I hope to have some time this week to actually re-read the code and implement this :)

**Comment by quba42** [comment]
@pieterlexis I am afraid this was automatically closed when I removed the `master` branch. Can you open a new PR against the `main` branch?

**Comment by pieterlexis** [comment]
> @pieterlexis I am afraid this was automatically closed when I removed the `master` branch. Can you open a new PR against the `main` branch?

I would, if it were not so incomplete. I'll see if I can find some time for it soon

**Pull request closed without merging**